apisix/plugins/request-validation.lua

```diff
@@ -94,11 +87,11 @@ function _M.rewrite(conf)
         if not body then
             local filename = ngx.req.get_body_file()
             if not filename then
-                return 500
+                return 500, conf.rejected_msg or nil
```

Comment:
```suggestion
                return 500, conf.rejected_msg
```

The default value of conf.rejected_msg is nil